src/ImportExport.ts

```diff
@@ -424,22 +424,25 @@ export const promocodes = async (input: string | null) => {
     } else if (input === 'time') {
         const availableUses = timeCodeAvailableUses();
         if (availableUses === 0) {
-            return Confirm(`
+            return Alert(`
             If you imported a save, you cannot use this code for 15 minutes to prevent cheaters.
```

Comment:
Here, fraudster measures aren't really working.
Should I take measures like gamble's localStorage?